nodes/text/mn_split_text.py

```diff
@@ -17,7 +17,8 @@ class mn_SplitText(Node, AnimationNode):
 	def splitTypeChanges(self, context):
 		self.setHideProperty()
 	
-	splitType = bpy.props.EnumProperty(name = "Split Type", default = "Custom", items = splitTypes, update = splitTypeChanges)
+	splitType = bpy.props.EnumProperty(name = "Split Type", default = "Regexp", items = splitTypes, update = splitTypeChanges)
+	keepDelimiters = bpy.props.BoolProperty(default=False)
```

Comment:
I agree for consistency, but it's against PEP 8 guidelines. Maybe one day you (or me or someone else) should reformat the code to adhere to PEP 8? :)